modules/configuring-ovnk-additional-networks.adoc

```diff
@@ -13,7 +13,7 @@ The {openshift-networking} OVN-Kubernetes network plugin allows the configuratio
 Pod and multi-network policy creation might remain in a pending state until the OVN-Kubernetes control plane agent in the nodes processes the associated `network-attachment-definition` CRD.
 ====
 
-You can configure an OVN-Kubernetes additional network in either _layer 2_ or _localnet_ topologies.
+You can configure an OVN-Kubernetes secondary network in either _layer 2_ or _localnet_ topologies.
```

Comment:
After 4.18 UDN release, layer3 is supported for OVN-Kubernetes secondary network. I am not sure whether you describe this in other modules.

Reply:
Added and dropped descriptions with a pointer to the UserDefinedNetwork and NetworkAttachmentDefinition support matrix table.